systemtest/src/test/java/io/strimzi/systemtest/TopicST.java

```diff
@@ -164,6 +164,38 @@ void testBigAmountOfTopicsCreatingViaKafka() {
         }
     }
 
+    @Test
+    void testTopicModificationOfReplicationFactor() {
+        String topicName = "topic-with-changed-replication";
+
+        testMethodResources().kafkaEphemeral(CLUSTER_NAME, 2, 1).done();
+
+        testMethodResources().topic(CLUSTER_NAME, topicName)
+                .editSpec()
+                    .withReplicas(2)
+                .endSpec()
+                .done();
+
+        TestUtils.waitFor("Waiting to " + topicName + " to be ready", Constants.GLOBAL_POLL_INTERVAL, Constants.TIMEOUT_FOR_TOPIC_CREATION,
+            () ->  testMethodResources().kafkaTopic().inNamespace(NAMESPACE).withName(topicName).get().getStatus().getConditions().get(0).getStatus().equals("True")
```

Comment:
Sorry I keep commenting on this :-o ... I think you should check for `type` being `Ready`. Even a failure would have `status` set to `True`

Reply:
fixed :)